store/dynamodb/dynamodb_test.go

```diff
@@ -28,9 +28,8 @@ func TestRegister(t *testing.T) {
 	assert.NoError(t, err)
 	assert.NotNil(t, kv)
 
-	if ddb, ok := kv.(*DynamoDB); !ok {
+	if _, ok := kv.(*DynamoDB); !ok {
 		t.Fatal("Error registering and initializing DynamoDB")
-		ddb.createTable()
```

Comment:
This is used to create the tables for testing, why remove it?

Reply:
the line before is a `t.Fatal` then `ddb.createTable()` is never called.